MailChimp.Net/Configuration/MailChimpConfiguration.cs

```diff
@@ -1,17 +1,36 @@
 ﻿using System;
-using System.Configuration;
+using System.Linq;
 using MailChimp.Net.Interfaces;
+using Microsoft.Extensions.Configuration;
 
 namespace MailChimp.Net
 {
     public class MailChimpConfiguration : IMailChimpConfiguration
     {
-        public static int DefaultLimit => Common.DefaultLimit;
+        public MailChimpConfiguration()
+        {
+            var builder = new ConfigurationBuilder()
```

Comment:
I think the standard now is to have a ``` IOptions< MailChimpOptions > ``` using ``` Microsoft.Extensions.Options ``` in order for it to work with how they are thinking with appsettings